wp-content/themes/citylimits/functions.php

```diff
@@ -362,3 +362,13 @@ function citylimits_newsletter_form_popover() {
 	get_template_part( 'partials/newsletter-signup', 'maincolumn' );
 	return ob_get_clean();
 });
+
+/**
+ * Register custom menu locations for the main nav and sticky mobile navs
+ */
+function register_citylimits_menu_locations() {
+	register_nav_menu( 'languages-menu', __( 'Languages Menu' ) );
+	register_nav_menu( 'mobile-sticky-menu', __( 'Mobile Sticky' ) );
+	register_nav_menu( 'special-projects-secondary-menu', __( 'Special Projects Secondary Navigation Menu' ) ) ;
```

Comment:
Do we want to deregister Largo's default menus here? https://github.com/INN/largo/blob/v0.6.4/functions.php#L218-L225

Reply:
@benlk Good idea. Updated in https://github.com/INN/umbrella-citylimits/pull/65/commits/c34aeabf7ae0b9936a67f8805846083e7515cd47